site/content/docs/5.0/components/popovers.md

```diff
@@ -249,9 +249,9 @@ Note that for security reasons the `sanitize`, `sanitizeFn`, and `allowList` opt
     </tr>
     <tr>
       <td><code>fallbackPlacements</code></td>
-      <td>string | array</td>
-      <td><code>'flip'</code></td>
-      <td>Allow to specify which position Popper will use on fallback. For more information refer to
+      <td>array</td>
+      <td><code>['top', 'right', 'bottom', 'left']</code></td>
+      <td>Define fallback placements by providing a list of placements in array (indexed by preferences). For more information refer to
```

Comment:
I don't quite get what does this mean exactly? "indexed by preferences" 😕

Reply:
For example, if you put: `['top', 'right']`, Popper will place the tooltip on top first and if the tooltip does not fit on the top then on the right side.

Reply:
We need to reword this maybe. @mdo: any suggestions?

Reply:
"indexed by preferences" > "in order of preference" ?